src/SFML/Window/OSX/SFWindowController.h

```diff
@@ -32,13 +32,10 @@
 ////////////////////////////////////////////////////////////
 /// Predefine some classes
 ////////////////////////////////////////////////////////////
-namespace sf
-{
-namespace priv
+namespace sf::priv
 {
 class WindowImplCocoa;
 }
-}
```

Comment:
`// namespace sf::priv` ?

Reply:
Not even clang-format-15 adds this comment, perhaps because the namespace is so small. Do we really need to add it here when the namespace is four lines up?

Reply:
It's okay, just noticed the inconsistency, but wasn't sure if it was an oversight or not, thus the question mark 😄